packages/combobox/src/LionCombobox.js

```diff
@@ -180,7 +182,6 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
    * @configure FormControlMixin
    * Will tell FormControlMixin that a11y wrt labels / descriptions / feedback
    * should be applied here.
-   * @protected
```

Comment:
This should be protected right?

Reply:
Yeah, definitely. I think I did this for consistency throughout the code base. Later we should start from the ground up and make all occurrences protected

Reply:
See https://github.com/ing-bank/lion/pull/1328, all fixed :)